### PR TITLE
[GraphAr] Open the graph test CI and fix GraphAr util namespace bug

### PR DESCRIPTION
**Pull request by acezen**
Signed-off-by: aceze <qiaozi.zwb@alibaba-inc.com>

Committed-by: aceze from Dev container

What do these changes do?
-------------------------

as title

Related issue number
--------------------

Fixes #issue number

### CODE REVIEW

**Comment by acezen** [comment]
cc/ @sighingnow @dashanji

File: .github/workflows/build-test-graph.yml [hunk 1]
```diff
@@ -19,12 +19,12 @@ on:
...
       - main
     paths:
-      - 'modules/graph'
+      - 'modules/graph/**'
   pull_request:
     branches:
       - main
     paths:
-      - 'modules/graph'
+      - 'modules/graph/**'
```
Comment (sighingnow) [inline .github/workflows/build-test-graph.yml:27]: Fixes for fixing.

**Review by sighingnow — APPROVED** [review]

File: .github/workflows/build-test-graph.yml [hunk 2]
```diff
@@ -105,13 +105,14 @@ jobs:
...
-                              libparquet-dev=11.0.0-1 \
-                              libplasma-dev=11.0.0-1 \
-                              libarrow-cuda-dev=11.0.0-1
+          sudo apt install -y libarrow-dev=12.0.1-1 \
+                              libarrow-dataset-dev=12.0.1-1 \
+                              libparquet-dev=12.0.1-1 \
+                              libarrow-acero-dev=12.0.1-1 \
+                              libarrow-flight-dev=12.0.1-1 \
+                              libgandiva-dev=12.0.1-1 \
+                              libparquet-dev=12.0.1-1 \
```
Comment (songqing) [inline .github/workflows/build-test-graph.yml:114]: parquet is duplicated, change to plasma?

Comment (sighingnow) [reply .github/workflows/build-test-graph.yml:114]: Good catch.

We don't require plasma.

Comment (sighingnow) [reply .github/workflows/build-test-graph.yml:114]: @acezen would you mind submitting a pull request for a quick fix?

**Pull request merged**